src/util/StreamLink.cpp

```diff
@@ -205,6 +209,13 @@ void openStreamlink(const QString &channelURL, const QString &quality,
 
 void openStreamlinkForChannel(const QString &channel)
 {
+    static const QString INFO_TEMPLATE("Opening %1 in Streamlink ...");
+
+    auto *currentPage = dynamic_cast<SplitContainer *>(
+        getApp()->windows->getMainWindow().getNotebook().getSelectedPage());
+    currentPage->getSelectedSplit()->getChannel()->addMessage(
+        makeSystemMessage(INFO_TEMPLATE.arg(channel)));
```

Comment:
`getSelectedPage()` and `getSelectedSplit()` can reasonably return a `nullptr`, so we should check for that